lerna.json

```diff
@@ -3,6 +3,5 @@
     "packages/*"
   ],
   "version": "0.1.1-alpha.41",
-  "npmClient": "pnpm",
-  "useWorkspaces": true
```

Comment:
Moest useWorkspaces ook weg?

Reply:
yes, versie 7 is dit niet meer nodig. Heb wel nog even packages in de package.json toegevoegd ook al lijkt dat niet nodig te zijn. 

The "useWorkspaces" option has been removed. By default lerna will resolve your packages using your package manager's workspaces configuration. Alternatively, you can manually provide a list of package globs to be used instead via the "packages" option in lerna.json.

Reply:
Oké top!